src/web/components/ParticipantRequests/ApprovedParticipantItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { ParticipantRequestDTO } from '../../../api/routers/participantsRouter';
+
+import './ParticipantRequestItem.scss';
+
+type ApprovedParticipantProps = {
+  participant: ParticipantRequestDTO;
```

Comment:
should this be `ParticipantRequestDTO` or `ParticipantDTO`?

Reply:
I'm going to make a note to combine these when I come back to this.